recipes/eventio/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "eventio" %}
+{% set version = "1.4.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 8f07de50ef4b06e1237ffbcbcd748fee71badf0719cf23c79a4425cd4194f8c1
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+  host:
+    - python
+    - pip
+    - numpy >=1.17
```

Comment:
If you need to enforce at least 1.17 don't use the >=, otherwise it will get the latest available only. Ideally you should not pin here and let conda-forge defaults be used. (I guess that our current default is 1.16, so that may not work, worth a try though.)